### ISSUE TITLE
Negative numbers sending using socket.emit

### ISSUE THREAD

**Issue by sliusar** [miguelgrinberg/python-socketio#676]
Hi,

I have a Pyramid-based webserver, which employs socketio to pass data to and from front-end. It was working fine before with Python 2.7 (using ancient python-socketio version 0.3.5 and sokcetio.js 0.9.6), but I had to upgrade and now I encounter an issue with negative values being sent from the web page to the Python backend. Whenever I try to send a negative value (-1, -1.5, array of values and at least one is negative) via `socket.emit("foo", value)`, I get the following exception:

```
message handler error
Traceback (most recent call last):
  File "/home/mgr/.local/lib/python3.6/site-packages/engineio/server.py", line 596, in _trigger_event
    return self.handlers[event](*args)
  File "/home/mgr/.local/lib/python3.6/site-packages/socketio/server.py", line 738, in _handle_eio_message
    pkt = packet.Packet(encoded_packet=data)
  File "/home/mgr/.local/lib/python3.6/site-packages/socketio/packet.py", line 40, in __init__
    self.attachment_count = self.decode(encoded_packet)
  File "/home/mgr/.local/lib/python3.6/site-packages/socketio/packet.py", line 83, in decode
    raise ValueError('too many attachments')
ValueError: too many attachments
2021-04-18 21:26:09,953 ERROR engineio.server [server.py:598] message handler error
Traceback (most recent call last):
  File "/home/mgr/.local/lib/python3.6/site-packages/engineio/server.py", line 596, in _trigger_event
    return self.handlers[event](*args)
  File "/home/mgr/.local/lib/python3.6/site-packages/socketio/server.py", line 738, in _handle_eio_message
    pkt = packet.Packet(encoded_packet=data)
  File "/home/mgr/.local/lib/python3.6/site-packages/socketio/packet.py", line 40, in __init__
    self.attachment_count = self.decode(encoded_packet)
  File "/home/mgr/.local/lib/python3.6/site-packages/socketio/packet.py", line 83, in decode
    raise ValueError('too many attachments')
ValueError: too many attachments
```

python-socketio version 5.2.0

I tried with two versions of socket.io.min.js: v3.0.3 and v4.0.1, in both cases packet.py throws this exception.

**Comment by miguelgrinberg** [comment]
This is fixed in python-socketio 5.2.1, please upgrade.

**Comment by sliusar** [comment]
Indeed, works with 5.2.1, thanks for the prompt reply!
